A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_ECAMGauge.js

```diff
@@ -352,7 +354,7 @@ var A320_Neo_ECAM_Common;
             }
             if (this.currentValueText != null) {
                 this.currentValueText.setAttribute("class", style);
-                if (this.currentValuePrecision == 0) {
+                if (this.uppercam == true) {
```

Comment:
You don't need to check for `true` wit `==` true, even tho `===` would be "more correct"
```js
if (this.upperecam) {
```

Reply:
That's true!